src/content/docs/cheerpj3/13-examples/04-java-browser.mdx

```diff
@@ -0,0 +1,166 @@
+---
+title: Java Browser (Networking)
+description: Run a Java application with Internet connection
+---
+
+In this tutorial, we will take you step by step on how to use CheerpJ and Tailscale to run a simple Browser client application written in Java. This tutorial aims to give a basic understanding of how to set wider internet access to a Java application that runs with CheerpJ.
+
+If you are not familiar with how CheerpJ and Tailscale work together, we invite you to take a look at our [Networking guide].
+
+## Pre-requisites
+
+- Download the application JAR [here](/cheerpj3/examples/SwingHTMLBrowser.jar)
+- [Node.js](https://nodejs.org/en/) (>= 18)
+- A simple HTTP server to host your page locally.
+- A text editor to create and edit an HTML file.
+- A modern browser like Chrome, Firefox or Safari.
+- A Tailscale account (explanation later in this tutorial)
+
+## 1. The Java application
+
+Let's start by reviewing the Java application we will be testing. This is a simple Swing HTML browser written in Java. You can run this JAR locally if you have Java installed on your machine, but feel free to skip this step.
+
+Functionality is simple, type the full URL of a website in the input field (e.g. `https://www.github.com/`) and press enter to load the page. This is a very basic example so the website will not render as in a modern browser but you will be able to see the hypertext.
+
+<table>
```

Comment:
Using a table for layout isn't the best- better to use a flex row that wraps on small widths. And don't hardcode the image widths